lib/xcpretty/formatters/formatter.rb

```diff
@@ -47,6 +47,8 @@ def format_test_suite_started(name);                      EMPTY; end
     def format_test_summary(message, failures_per_suite);     EMPTY; end
     def format_touch(file_path, file_name);                   EMPTY; end
     def format_tiffutil(file);                                EMPTY; end
+    def format_write_file(file);                              EMPTY; end
```

Comment:
Unused method argument - `file`. If it's necessary, use `_` or `_file` as an argument name to indicate that it won't be used. You can also write as `format_write_file(*)` if you want the method to accept any arguments but don't care about them.<br>Use empty lines between defs.<br>Unnecessary spacing detected.